packages/plugin-ext/src/plugin/types-impl.ts

```diff
@@ -1728,12 +1728,14 @@ export class Task {
         if (this.taskExecution instanceof ProcessExecution) {
             Object.assign(this.taskDefinition, {
                 type: 'process',
-                id: this.taskExecution.computeId()
+                id: this.taskExecution.computeId(),
+                taskType: this.taskDefinition!.type
```

Comment:
For my own information, are we certain that `taskDefinition` will not be `undefined`?

Reply:
You are right. resolved as part of https://github.com/theia-ide/theia/pull/5542. Thank you for the review !